src/Shared/API_Util/GPUData.hpp

```
@@ -0,0 +1,56 @@
+
+    float constant = 1.0f;
+    float linear = 0.09f;
+    float quadratic = 0.032f;
+    float _pad2 = 0.0f;
+
+    glm::vec3 ambient = glm::vec3(1.0f, 1.0f, 1.0f);  float _pad3 = 0.0f;
+    glm::vec3 diffuse = glm::vec3(1.0f, 1.0f, 1.0f) ;  float _pad4 = 0.0f;
+    glm::vec3 specular = glm::vec3(.05f, .05f, .05f); float _pad5 = 0.0f;
+};
+// TODO: Fix light
+struct alignas(16) SpotLightGPU : LightBaseGPU
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: cosCutOff, cosOuterCutOff

(cppcoreguidelines-pro-type-member-init)